src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/LanguageClient/Endpoints/RazorCustomMessageTarget.cs

```diff
@@ -200,6 +201,21 @@ private async Task<SynchronizedResult<TVirtualDocumentSnapshot>> TrySynchronizeV
             _logger.LogDebug($"{(result.Synchronized ? "Did" : "Did NOT")} synchronize for {caller}: Version {requiredHostDocumentVersion} for {result.VirtualSnapshot?.Uri}");
         }
 
+        //if (requiredHostDocumentVersion == 1 &&
+        //    result.Synchronized &&
+        //    result.VirtualSnapshot is not null &&
+        //    range is not null)
+        //{
+        //    // If we're at version 1, and we're synced, lets make sure the buffer has enough lines to handle the request
+        //    if (range.Start.Line > result.VirtualSnapshot.Snapshot.LineCount ||
+        //        range.End.Line > result.VirtualSnapshot.Snapshot.LineCount)
+        //    {
+        //        Debug.Fail("It worked!");
+        //        _logger.LogWarning($"Requested line ({range.Start.Line} or {range.End.Line}) is out of bounds for {result.VirtualSnapshot.Uri}. Sync failed.");
+        //        return new SynchronizedResult<TVirtualDocumentSnapshot>(false, null);
+        //    }
+        //}
+
```

Comment:
I'm just noting this commented code in case it doesn't get removed in a later commit.